src/nunit.analyzers/Extensions/AttributeArgumentSyntaxExtensions.cs

```diff
@@ -55,7 +56,7 @@ internal static class AttributeArgumentSyntaxExtensions
             }
             else
             {
-                var targetType = GetTargetType(target);
+                ITypeSymbol? targetType = GetTargetType(target);
```

Comment:
The need for this is disappearing (has disappeared already?) in a newer version of the C# compilers, treated as a bugfix. The inferred type of `var` is always tracked as nullable now for the purpose of future assignments. This doesn't lose the tracked not-null state that the initial value gets.

Reply:
Yeah, it compiled perfectly fine for me with VS preview, but failed in Appveyor build.

Reply:
Good to know.